staging/traefik-forward-auth/templates/hooks.yaml

```diff
@@ -77,7 +77,7 @@ spec:
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: "{{.Release.Name}}-session-key-pre-install"
+  name: "{{.Release.Name}}-sesh-key-pre-install"
```

Comment:
`sesh` we're so hip 😆

Reply:
![image](https://user-images.githubusercontent.com/4969231/133332502-d446a89a-a15c-468b-b3fb-5fb5e7cdefa1.png)

Reply:
LOL